blobstore/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-interface-blobstore"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
This should be a minor bump to `0.2.0`

Reply:
Unless `0.1.0` was published with wasmbus-rpc `0.8`, in which case disregard

Reply:
done